### PR TITLE
Fix asset combiner

### PR DESCRIPTION
**Pull request by cleverer**
Opening a pull request for easier discussion of #3264.

I'm not that familiar with the inner workings of October, I think we should get an error, if we have a path like `~/themes/demo/assets/test.css`but in my tests it did work.

### CODE REVIEW

File: modules/system/traits/AssetMaker.php [hunk 1]
```diff
@@ -142,7 +146,11 @@ public function addJs($name, $attributes = [])
     public function addCss($name, $attributes = [])
     {
         if (is_array($name)) {
-            $name = $this->combineAssets($name);
+	        $assetPath = $this->assetPath;
+	        if (!starts_with($assetPath, [base_path()])) {
```
Comment (LukeTowers) [inline modules/system/traits/AssetMaker.php]: Can you put this in a helper method, something like `getLocalPath()`?

Comment (cleverer) [reply modules/system/traits/AssetMaker.php]: Sure thing!

**Comment by LukeTowers** [comment]
Thanks @cleverer. Could you add some unit tests to test the following cases:
```php
$this->assetPath = $defaultPublicPath;
$this->assetPath = $pathWithSymbols // both types of symbols
$this->assetPath = $pathThatAlreadyIncludesBasePath
```

**Comment by LukeTowers** [comment]
@cleverer the second approach works for me, the public path would be generated by https://github.com/octobercms/october/blob/master/modules/backend/classes/WidgetBase.php#L54

**Comment by LukeTowers** [comment]
Awesome, thanks very much @cleverer!

**Pull request merged**